spec/support/http_cache/vcr/aip_download_and_swift_upload.yml

```diff
@@ -1,22 +1,76 @@
 ---
 http_interactions:
+- request:
+    method: post
+    uri: http://localhost:3000/auth/system?email=ditech@ualberta.ca&password=correct%20horse%20battery%20staple
```

Comment:
I feel like password/email should be within the post body, not as query params 🤔 . This could be a potential security issue (rollbar for example, if this errored out, rollbar will expose this url on its dashboard and elsewhere which now includes the systems username/password)

Reply:
You are right about where to send the data! This is a post call so I figured the data I was sending would have been sent as part of the body. Will need to double check how the request is being made.